app/src/main/java/com/vci/vectorcamapp/core/presentation/search/SearchTextField.kt

```diff
@@ -15,7 +16,12 @@ fun SearchTextField(
     placeholder: String,
     modifier: Modifier = Modifier,
     isSingleLine: Boolean = true,
-    onSearchSubmitted: (() -> Unit)? = null
+    isTooltipVisible: Boolean = false,
+    onSearchSubmitted: (() -> Unit)? = null,
+    onTooltipShow: (() -> Unit)? = null,
+    onTooltipDismiss: (() -> Unit)? = null,
+    tooltipButtonText: String = "Tap to learn more about searching",
```

Comment:
Question: Do we need this parameter? I think we can just keep it as static text for the button content because the purpose of this is to display on the search text field right? I don't see many custom texts one would want to have.